src/core/server/locales/en-US/common.ftl

```diff
@@ -12,4 +12,11 @@ comment-count =
     *[other] Comments
   }</span>
 
+comment-counts-ratings-and-review =
```

Comment:
What do you think about naming this `comment-counts-ratings-and-reviews` with plural `reviews` for consistency for how we refer to ratings and reviews elsewhere in the code?

Reply:
Renamed